.ci/scripts/shellcheck

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
is this required?

Reply:
https://github.com/elastic/apm-server/pull/2498#discussion_r320318790

Reply:
IIUC, this particular script is not used in this repo. If so, where about?